pandas/tests/libs/test_groupby.py

```diff
@@ -0,0 +1,237 @@
+import numpy as np
```

Comment:
how about a counter proposal. can we make a libs/ sub-dir in pandas/tests/groupby (and similar for reshape)?

I find that if these are in different trees then people have no idea where to put the tests.

Reply:
sure

Reply:
updated+greenish